### PR TITLE
Added links to roadmap

### LINKED ISSUE #4042: Add links to roadmap
### Description

I want to add links to the roadmap so that anyone can directly click on the respective topic and go to that folder, instead of browsing through the whole repository.

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Others

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by rajmadhu0406**
<hr>

## Description 📜

Added links to roadmap by editing the readme.md file.
Fixes #4042 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [x] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/ReadMe.md
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/ReadMe.md:8]: Hey, Please add all the links in one format.
For eg: `[DNS & Working of DNS](./DNS%20and%20Working%20of%20%DNS.md)`

Use `%20` for spaces.

File: Web_Development/FrontEnd/ReadMe.md
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/ReadMe.md:35]: Same comments here.

File: Web_Development/FrontEnd/ReadMe.md
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/ReadMe.md:42]: Same comments here.

File: Web_Development/FrontEnd/ReadMe.md
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/ReadMe.md:50]: same comments here.

**Review by Aayushi-Mittal — CHANGES_REQUESTED** [review]
@rajmadhu0406 Please do the requested changes. Rest looks good.

File: Web_Development/FrontEnd/ReadMe.md
Comment (rajmadhu0406) [reply Web_Development/FrontEnd/ReadMe.md:50]: okay

**Comment by rajmadhu0406** [comment]
@Aayushi-Mittal  please take a look, I have done the required.

**Review by Aayushi-Mittal — APPROVED** [review]
Perfect! Please merge this PR @Ayan-thecodeking.

**Review by SauravMukherjee44 — APPROVED** [review]

**Review by heet-kumar — APPROVED** [review]
LGTM !! perfect 👍 lets wait for the supervisor approval .

**Pull request merged**
